internal/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -73,13 +76,12 @@ RUN yum clean all && rm -rf /var/cache/yum/* \
  && pip3 install --no-cache-dir --ignore-installed ipaddress \
       ansible-runner==1.3.4 \
       ansible-runner-http==1.0.0 \
-      openshift==0.9.2 \
+      openshift~=0.10.0 \
```

Comment:
Add entry in the changelog over the upgrade of openshift version.

Reply:
:+1: done